src/main/java/seedu/address/logic/commands/TaskCommand.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MODULE;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.task.Task;
+
+/**
+ * Adds a task to the task list.
+ */
+public class TaskCommand extends Command {
```

Comment:
Not too sure but was wondering if AddTaskCommand would be a more suitable name for this class